images/sdn/Dockerfile.mine

```diff
@@ -0,0 +1,55 @@
+FROM registry.svc.ci.openshift.org/ocp/builder:golang-1.10 AS builder
+WORKDIR /go/src/github.com/openshift/origin
+COPY . .
+RUN make build WHAT=" \
+    cmd/openshift-sdn \
+    cmd/sdn-cni-plugin \
+    vendor/github.com/containernetworking/plugins/plugins/ipam/host-local \
+    vendor/github.com/containernetworking/plugins/plugins/main/loopback \
+    "
+
+FROM registry.svc.ci.openshift.org/ocp/4.0:base AS RHEL7
+COPY mirror/ocp-4.0-default.repo /etc/yum.repos.d/ocp-4.0-default.repo
+COPY mirror/ops-mirror.pem /etc/yum.repos.d/ops-mirror.pem
+COPY --from=builder /go/src/github.com/openshift/origin/_output/local/bin/linux/amd64/openshift-sdn /usr/bin/
+COPY --from=builder /go/src/github.com/openshift/origin/_output/local/bin/linux/amd64/sdn-cni-plugin /opt/cni/bin/openshift-sdn
+COPY --from=builder /go/src/github.com/openshift/origin/_output/local/bin/linux/amd64/loopback /opt/cni/bin/
+COPY images/sdn/scripts/* /usr/sbin/
+COPY --from=builder /go/src/github.com/openshift/origin/_output/local/bin/linux/amd64/host-local /opt/cni/bin/
+RUN INSTALL_PKGS=" \
+      openvswitch2.11 container-selinux socat ethtool iptables nmap-ncat \
+      libmnl libnetfilter_conntrack conntrack-tools \
+      libnfnetlink iproute bridge-utils procps-ng openssl \
+      binutils xz sysvinit-tools dbus \
+      " && \
+    rm /etc/yum.repos.d/content.repo && \
+    yum install -y --setopt=tsflags=nodocs $INSTALL_PKGS && \
+    mkdir -p /etc/sysconfig/cni/net.d && \
```

Comment:
do you know why this is needed? I don't remember /etc/sysconfig/cni/net.d in the daemonset mounts. I have been using /etc/cni/net.d (I am trying to figure out how all this works)